### ISSUE TITLE
Logo for the project + website

### ISSUE THREAD

**Issue by arupgsh** [BioinfoNet/Affordabee#9]
It will be great if someone can design a logo that represents the motto of the project.

**Comment by kipkurui** [comment]
Absolutely. Working on it, and also getting help from designers on the Sprint

**Comment by kipkurui** [comment]
<img width="330" alt="image" src="https://user-images.githubusercontent.com/3762127/92081848-6281b380-edcc-11ea-995f-e7fa10e9c390.png">

**Comment by kipkurui** [comment]
Thoughts?

**Comment by Eliz-Oliver1** [comment]
I think it looks great! I like the simplicity. I have zero experience with logo design but came up with this if anyone wants to play around with the concept...
[Logo idea.pptx](https://github.com/BioinfoNet/Affordabee/files/5167465/Logo.idea.pptx)

**Comment by Jennifer-M1** [comment]
I like it alott.

**Comment by Jennifer-M1** [comment]
Are we expecting more logo options?

**Comment by kipkurui** [comment]
![image](https://user-images.githubusercontent.com/3762127/92098209-9ca98000-ede1-11ea-9c6e-3846f7cbe575.png)

**Comment by kipkurui** [comment]
![image](https://user-images.githubusercontent.com/3762127/92109553-e4380800-edf1-11ea-8cc8-022647bba363.png)
